### ISSUE TITLE
Multiline quotes

### ISSUE THREAD

**Issue by DanInSpace104** [bouncepaw/mycorrhiza#51]
Now you can't quote something like quatrain, cause there is no linebreak syntax and multiline quotes. My syntax suggestions:

1. Block
```
quote {
Lorem ipsum dolor sit amet, 
consectetur adipiscing elit,
sed do eiusmod tempor incididunt
ut labore et dolore magna aliqua.
}
```
2. Line break syntax (maybe other symbol)
```
> Lorem ipsum dolor sit amet, \\n consectetur adipiscing elit, \\n sed do eiusmod tempor incididunt \\n ut labore et dolore magna aliqua.
```
3. Merge quotes. Different quotes separates with empty line.
```
> Lorem ipsum dolor sit amet, 
> consectetur adipiscing elit,
> sed do eiusmod tempor incididunt
> ut labore et dolore magna aliqua.
```

**Comment by bouncepaw** [comment]
@DanInSpace104 Yeah, such problem exists. The third option will be done, in fact, it is currently being developed.

I also like the first option, but instead of quote, I'm thinking of > itself. Like that:


> { 
Lorem ipsum dolor sit amet,  
consectetur adipiscing elit, 
sed do eiusmod tempor incididunt 
ut labore et dolore magna aliqua. 
}

**Comment by DanInSpace104** [comment]
@bouncepaw can you wrap your example with code syntax? so we will see ">" symbol )

**Comment by bouncepaw** [comment]
Done.

**Comment by bouncepaw** [comment]
The third option has been implemented in 1.2.

Here's what one can write:

```
> multiline quote
>
> paragraph 2
> > nest
> => Rocket
```
